treeherder/perf/models.py

```diff
@@ -447,6 +447,11 @@ class PerformanceAlert(models.Model):
         help_text="t value out of analysis indicating confidence " "that change is 'real'",
         null=True,
     )
+    noise_profile = models.CharField(
```

Comment:
I think it could be useful to define the [choices](https://docs.djangoproject.com/en/3.2/ref/models/fields/#choices) for the field `noise_profile` :

- `SKEWED` - samples are heavily to one side of the mean
- `OUTLIERS` - there are more outliers than would be expected from normal distribution
- `MODAL` - few samples are near the mean (probably bimodal)
- `OK` - no egregious deviation from normal
- `N/A` - not enough data to make a conclusion (aka OK)